test/trento_web/controllers/v1/suse_manager_controller_test.exs

```diff
@@ -139,4 +140,50 @@ defmodule TrentoWeb.V1.SUSEManagerControllerTest do
         |> assert_schema("PatchesForPackagesResponse", api_spec)
     end
   end
+
+  describe "retrieve errata details" do
+    test "should return errata details", %{conn: conn, api_spec: api_spec} do
+      insert_software_updates_settings()
+
+      advisory_name = Faker.Pokemon.name()
+
+      %{errataFrom: errata_from} = errata_details = build(:errata_details)
+
+      expect(Trento.SoftwareUpdates.Discovery.Mock, :get_errata_details, 1, fn _ ->
+        {:ok, errata_details}
+      end)
+
+      resp =
+        struct(
+          ErrataDetailsResponse,
+          errata_details
+          |> Map.drop([:errataFrom])
+          |> Map.put(:errata_from, errata_from)
+        )
+
+      ^resp =
```

Comment:
I think you could just match `%ErrataDetailsResponse{errata_from: errata_from}`, maybe also add the synopsis explicitly :+1: